src/Interpreters/AsynchronousMetrics.cpp

```diff
@@ -123,9 +123,9 @@ void AsynchronousMetrics::openSensors()
         {
             LOG_WARNING(
                 &Poco::Logger::get("AsynchronousMetrics"),
-                "Thermal monitor '{}' exists but could not be read, error {}.",
+                "Thermal monitor '{}' exists but could not be read: {}.",
                 thermal_device_index,
-                e.getErrno());
+                errnoToString(e.getErrno()));
```

Comment:
The code in https://github.com/ClickHouse/ClickHouse/pull/33639 did not print error messages properly. It was printing something incomprehensible like "error 123" instead of a proper message.